### ISSUE TITLE
Regression: Power Cycle in debugger doesn't reload rom file

### ISSUE THREAD

**Issue by veremenko-y** [SourMesen/Mesen#871]
Behaviour on 0.9.9
If rom is changed, Power Cycle in Debugger doesn't load new ROM code
Behaviour on 0.9.8
If rom is changed, Power Cycle in Debugger loads updated ROM file.

**Comment by bunder2015** [comment]
This is intentional, if you wish to reload the ROM you can do it from the main window, under the File menu.

**Comment by veremenko-y** [comment]
> This is intentional, if you wish to reload the ROM you can do it from the main window, under the File menu.
I know I can. But now instead of using hotkey on the keyboard, I had to grab the mouse, switch the window, go trough menu, then click back on the debugger. Plus this change was non announced on the changelog, which I would consider breaking change.

**Comment by bunder2015** [comment]
Here is the commit that changed it... https://github.com/SourMesen/Mesen/commit/43811ae7acec0f46035b62db82c0e864ca9844ae

The reason it was changed was that it would cause the debugger to throw away changes when you had to reset/powercycle the game, which can make debugging and programming inside mesen a little tricky.

**Comment by veremenko-y** [comment]
>The reason it was changed was that it would cause the debugger to throw away changes when you had to reset/powercycle the game, which can make debugging and programming inside mesen a little tricky.
I respectfully disagree. From my perspective it makes researching/modifying existing ROM painful. What's the reasonable context to preserve local changes, if my ROM derives from the my own code? I'm not saying this should be reverted, but I really need an option or separate shortcut to work on my code. I wasted too much time debugging stale ROMs, because I forgot to reload it. I had to downgrade to previous version.

Shortcut requires to switch focus to main menu, which doesn't help much.
